src/dmd/frontend.h

```diff
@@ -5511,6 +5511,7 @@ class TypeNoreturn final : public Type
     const char* kind() const;
     TypeNoreturn* syntaxCopy();
     MATCH implicitConvTo(Type* to);
+    MATCH constConv(Type* to);
```

Comment:
And update mtype.h

Reply:
`TypeNoreturn` is currently missing from `mtype.h`. But i can add it too

Reply:
Done